_data/Kikemascote/Kikemascote.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Enrique Mascote 
+institution: UTCH  
+profile_pic: Kike.jpg 
+quote: Covid-19 me ha pelado la berengena y dos que tres morros mas tambien
```

Comment:
Hi Enrique,
This quote goes against the event Code of Conduct. We need you to change it before we can move forward with your application.

Reply:
let me fix that

```suggestion
quote: Let's get this work togheter
```